### DISCUSSION TITLE
Can we use material-tailwind on our eCommerce website with a CMS?

### DISCUSSION THREAD

**Discussion by r1y4h** [creativetimofficial/material-tailwind#658] [General]
On the pricing page, there is this FAQ:

**Can I create or sell templates or themes using Material Tailwind?**
_Our product cannot be used to create templates, UI kits, dashboards, or web builders that are distributed on your website or other marketplaces._

We are creating an ERP application that will have an eCommerce (with CMS built-in) application for our clients as part of the ERP suite. It's a multi-tenant application, some clients may want to have a different look (themes) and feel (content) on their ecommerce website. I believe a CMS is different from a web builder. But I have to ask if creating template like changing the theme colors, will violate your policy?
We're not building an application that competes with material-tailwind.

**Comment by alexandru-paduraru** [comment]
Hey @r1y4h thank you for your interest in working with our framework. As long as you don't do direct competition it's fine to use it as the framework for the CMS having also themes colors etc.

We would appreciate if you add credits somewhere so other people who build projects can use it too.

Hope this helps,
Alex
